<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BTChat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>600</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chatBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>721</width>
      <height>491</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>721</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Communication</string>
    </property>
    <widget class="QLabel" name="statusStaticLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>101</red>
           <green>101</green>
           <blue>101</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Unconnected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>30</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="localhostgroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>721</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Localhost</string>
    </property>
    <widget class="QLabel" name="deviceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>711</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Device:</string>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>711</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bt_addressLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>711</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Bluetooth address:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
